firmware/screen/src/main.cpp

```
@@ -195,6 +277,109 @@ static void refresh_ui() {
+    // LV_COLOR_16_SWAP=1 → bytes already in big-endian RGB565. The
+    // moononournation API name is "BeRGBBitmap" — feed straight in.
+    gfx->draw16bitBeRGBBitmap(area->x1, area->y1,
+                              reinterpret_cast<uint16_t*>(color_p),
+                              w, h);
+    lv_canvas_dirty = true;
+  }
+  lv_disp_flush_ready(drv);
+}
+
+static void lv_indev_read_cb(lv_indev_drv_t* /*drv*/, lv_indev_data_t* data) {
+  uint16_t x = 0, y = 0;
```

Clang (14.0.6)

[warning] variable 'x' is not initialized

(cppcoreguidelines-init-variables)